demo/index.html

```diff
@@ -221,32 +227,6 @@
       </demo-snippet>
 
       <h3>
-        Tabs can be used in a <code>paper-toolbar</code>.
```

Comment:
actually I get that question almost exclusively on iron-pages. Will request there instead